index.js

```diff
@@ -1,17 +1,41 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items = this.items.sort((a, b) => {
```

Comment:
sort() es un método que "muta" (modifica directamente) el array en el que se aplica, asi que ni hace falta almacenarlo de nuevo en la variable this.items